loader/vk_loader_platform.h

```diff
@@ -370,10 +362,10 @@ static inline const char *loader_platform_open_library_error(const char *libPath
 #endif
 }
 static inline void loader_platform_close_library(loader_platform_dl_handle library) {
-// We only want to remove dlclose if both Asan is active and LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING is set
-#if SHOULD_UNLOAD_DYNAMIC_LIBRARIES == 1
-    dlclose(library);
+#if defined(LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING)
```

Comment:
Nit: Could we have info log messages at least indicating that library unloading is disabled in this and the next function?  Just for tracking purposes.

Reply:
Added logging